TotalCrossVM/CMakeLists.txt

```diff
@@ -0,0 +1,620 @@
+cmake_minimum_required(VERSION 3.13.5)
```

Comment:
Has this version been validated? If we don't need 3.13, then we should reduce the requirement as we make it easier for people to build on their host distributions.

Reply:
How so validated? We need these minimum version rquirement because we use some features of [cmake-generators(7) ](https://cmake.org/cmake/help/v3.13/manual/cmake-generators.7.html) that, theoretically, according to this documentation are part of CMake since version 3.13.x. The feature is related to [this doc](https://cmake.org/cmake/help/v3.13/generator/Visual%20Studio%209%202008.html). *And why is it relevant?* For me, it brings a slightly more *modular* experience, the generation for VS project became more similar to XCode generation. Both Win32 and WinCE (and their subprojects) are smoother to open into Visual Studio 2008 IDE.

Reply:
version required reduced to 3.5